src/app/pages/directoryservice/ldap/ldap.component.ts

```diff
@@ -98,86 +98,86 @@ export class LdapComponent {
     },
     {
       type : 'input',
       name : 'ldap_passwordsuffix',
       placeholder : T('Password Suffix'),
       tooltip: T('Enter a suffix to add to the password when it is added\
-                to the LDAP directory.')
+                  to the LDAP directory.')
     },
     {
       type : 'input',
       name : 'ldap_machinesuffix',
       placeholder : T('Machine Suffix'),
       tooltip: T('Enter a suffix to add to the system name when it is\
-                added to the LDAP directory.')
+                  added to the LDAP directory.')
     },
     {
       type : 'input',
       name : 'ldap_sudosuffix',
       placeholder : T('SUDO Suffix'),
       tooltip: T('Enter the suffix for LDAP-based users that need\
-                superuser access.')
+                  superuser access.')
     },
     {
       type : 'select',
       name : 'ldap_kerberos_realm',
       placeholder : T('Kerberos Realm'),
       tooltip: T('Select the realm created using the instructions in <a\
-                href="..//docs/directoryservice.html#kerberos-realms"\
-                target="_blank">Kerberos Realms</a>.'),
+                  href="..//docs/directoryservice.html#kerberos-realms"\
+                  target="_blank">Kerberos Realms</a>.'),
       options : []
     },
     {
       type : 'select',
       name : 'ldap_kerberos_principal',
       placeholder : T('Kerberos Principal'),
       tooltip: T('Select the location of the principal in the keytab\
-                created as described in <a\
-                href="..//docs/directoryservice.html#kerberos-keytabs"\
-                target="_blank">Kerberos Keytabs</a>.'),
+                  created as described in <a\
+                  href="..//docs/directoryservice.html#kerberos-keytabs"\
+                  target="_blank">Kerberos Keytabs</a>.'),
       options : []
     },
     {
       type : 'select',
       name : 'ldap_ssl',
       placeholder : T('Encryption Mode'),
-      tooltip: T('Authentication only functions when a <b>Certificate</b>\
-                is selected with either the <i>SSL</i> or <i>TLS</i>\
-                option.'),
+      tooltip: T('Authentication only functions when a Certificate\
+                  is selected with either the <i>SSL</i> or <i>TLS</i>\
+                  option.'),
       options : []
     },
     {
       type : 'select',
       name : 'ldap_certificate',
       placeholder : T('Certificate'),
       tooltip: T('Select the LDAP CA certificate. The certificate for the\
-                LDAP server CA must first be imported using the\
-                <b>System/Certificates</b> menu.'),
+                  LDAP server CA must first be imported using the\
+                  System/Certificates menu.'),
       options : []
     },
     {
       type : 'input',
       name : 'ldap_timeout',
       placeholder : T('LDAP timeout'),
       tooltip: T('Add more time in seconds if a Kerberos ticket time out\
```

Comment:
"LDAP timeout in seconds. Increase this value if a Kerberos ticket timeout occurs."